shared_memory/main.cpp

```diff
@@ -298,7 +304,17 @@ int main(int argc, char **argv)
 		}
 	}
 
-	if (load_all || load_bd) {
+	if(load_all || load_splash_factions) {
+		LogInfo("Loading splash factions");
+		try {
+			LoadSplashFaction(&database, hotfix_name);
```

Comment:
Should this be `&content_db`?

Reply:
I suppose so, I did write this before repositories existed.